dor/adapters/technical_metadata.py

```diff
@@ -79,13 +83,42 @@ def __str__(self):
         return ET.tostring(self.metadata, encoding="unicode")
 
 
+@dataclass
+class TextTechnicalMetadata:
+    mimetype: Mimetype
+    metadata: ET.Element
+    status: JHOVEStatus
+    valid: bool
+
+    @classmethod
+    def create(cls, file_path: Path) -> Self:
+        jhove_doc = JHOVEDoc.create(
+            file_path,
+            JHOVE_TEXT_METADATA_PROPERTY
+        )
+
+        return cls(
+            mimetype=Mimetype(jhove_doc.mimetype),
+            metadata=jhove_doc.technical_metadata,
+            status=JHOVEStatus(jhove_doc.status),
+            valid=jhove_doc.valid
+        )
+
+    @property
+    def metadata_mimetype(self) -> TechnicalMetadataMimetype:
+        return TechnicalMetadataMimetype.TEXTMD
+
+    def __str__(self):
+        return ET.tostring(self.metadata, encoding="unicode")
+
+
 class JHOVEDoc:
 
     @classmethod
     def create(cls, file_path: Path, metadata_property: str) -> Self:
         try:
             jhove_output = subprocess.run(
-                ["/opt/jhove/jhove", "-h", "XML", file_path],
+                ["/opt/jhove/jhove", "-h", "XML", "-c", "./etc/jhove.conf", file_path],
```

Comment:
I _think_ I like explicitly passing the JHOVE config here, rather than installing it at `opt/jhove/conf`, but happy to make the change if people feel differently.